### PR TITLE
Add missing NS flags and minor changes to the kserve demo

### PR DESCRIPTION
**Pull request by bdattoma**
## Description
Few changes in the kserve.md quick start:
- adding `-n $TEST_NS` in some lines (monitoring part not checked yet)
- removing annotation from SA because those are in the secret too
- add a note about using S3 storage different from minio
- add a step to enable http2 if not enabled in the cluster
- add expected model answer

## How Has This Been Tested?
following the quick start on a PSI cluster

## Merge criteria:

- [ ] The commits are squashed in a cohesive manner and have meaningful messages.
- [X] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [X] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: demo/kserve/Kserve.md [hunk 1]
```diff
@@ -6,6 +6,8 @@
 - CLI tools
   - oc cli
 
+- http2 enabled in the cluster
```
Comment (Jooho) [inline demo/kserve/Kserve.md:9]: Please add this too.
``` 
cluster-admin permission need
```

Comment (bdattoma) [reply demo/kserve/Kserve.md:9]: don't we need cluster admin permission for other steps too? like installing operators

File: demo/kserve/Kserve.md [hunk 2]
```diff
@@ -161,25 +162,53 @@ sed "s/<test_ns>/$TEST_NS/g" custom-manifests/service-mesh/peer-authentication-t
 ~~~
 
 ### Create Caikit ServingRuntime
-
+Before running the next line: if you are going to serve the model using CPU and not GPU, you need to set the following parameter in the runtime config (you find a comment in the YAML file too):
```
[the remark this answers is not in the document]
Comment (bdattoma) [reply demo/kserve/Kserve.md:165]: eheh you opted for the fancy solution at the end :P sure I will add this

File: demo/kserve/Kserve.md (hunk 1 above)
Comment (bdattoma) [reply demo/kserve/Kserve.md:9]: done!

File: demo/kserve/Kserve.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (bdattoma) [reply demo/kserve/Kserve.md:165]: done!

File: demo/kserve/Kserve.md (hunk 1 above)
Comment (Jooho) [reply demo/kserve/Kserve.md:9]: It is because of SMMR but it might need less power but just in case, it should be good to have strong permission.

Comment (bdattoma) [reply demo/kserve/Kserve.md:9]: no I mean to say that for running the entire quick start we should have the cluster admin permissions. Should we write it in the pre-requiresites rather?

Comment (Jooho) [reply demo/kserve/Kserve.md:9]: In order to install operators, yes cluster admin permission is needed but I realized some users have a cluster all required operators were installed. So I think it is better to write this down into pre-requisites.

Comment (bdattoma) [reply demo/kserve/Kserve.md:10]: done. However, we should document it more precisely in the API documentation which I strongly suggest to start authoring

**Pull request merged**